notebooks/Example_Notebook.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Again, please add axis labels and a title.</p><p> </p><p>What do you think about making this one green, instead of red?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lruhlen/ReviewNB_test/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I'd prefer blue to green.</p>